README.md

```diff
@@ -59,30 +59,30 @@ Creating machine...
 Starting local OpenShift cluster...
 OpenShift is available at https://192.168.99.100:8443.
 
-$ oc run hello-minishift --image=gcr.io/google_containers/echoserver:1.4 --port=8080 --expose --service-overrides='{"apiVersion": "v1", "spec": {"type": "NodePort"}}'
-service "hello-minishift" created
-deploymentconfig "hello-minishift" created
+$ oc create -f hello-openshift.json
```

Comment:
@jimmidyson That would a nice example but I would go with `hello-openshift` demo as quick start demo as `openshift/hello-openshift` is around `7.6 MB` and will enable users to start quickly.

However, `ruby-22-centos7` is around `440 MB` and may not be easy for users with less bandwidth.

Moreover, I found `ruby-hello-world` repo is not update with service type `NodePort` which may adds additional steps in the accessing application like exposing service.

Let me know what do you think.